plugins/woocommerce/includes/wc-template-functions.php

```diff
@@ -1238,9 +1238,17 @@ function woocommerce_template_loop_category_link_close() {
 	function woocommerce_taxonomy_archive_description() {
 		if ( is_product_taxonomy() && 0 === absint( get_query_var( 'paged' ) ) ) {
 			$term = get_queried_object();
+			
+			/**
+			 * Filters the archive's raw description on taxonomy archives.
+			 *
+			 * @param string $term_description
+			 * @param string $term
```

Comment:
1. I think this will be a `WP_Term` object rather than a string...
2. ...Though if we keep the filter in this location it may also be `null`. To that end, what do you think of moving the filter inside the `if ( $term ) { ... }` block? It feels like there is limited value in applying the filter if the term object could not be obtained (and the filtered value will anyway not be rendered in these circumstances).

Reply:
OK, if you're happy to make a change (update the type information in the docblock, and move the `apply_filters()` call so it isn't triggered unnecessarily), I think we'll be good to merge :+1: